lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -72,6 +72,11 @@ class OCI8Statement implements \IteratorAggregate, Statement
     protected $_paramMap = array();
 
     /**
+     * @var array
```

Comment:
This docblock needs additional documentation, describing why we are actually keeping a reference to the bound data (this is an oracle bug yadda yadda, horrible PHP7 migration path yadda yadda)